app/frontend/javascript/.eslintrc.js

```diff
@@ -1,18 +1,22 @@
 module.exports = {
   env: {
+    'jest/globals': true,
     browser: true,
+    commonjs: true,
     es6: true,
     jasmine: true,
-    'jest/globals': true,
+    node: true,
   },
   globals: {
     global: true,
+    SCAPE: true,
   },
   plugins: ['jest', 'vue'],
-  extends: ['eslint:recommended', 'plugin:vue/recommended', 'prettier'],
+  extends: ['eslint:recommended', 'plugin:vue/recommended', 'plugin:jest/recommended', 'prettier'],
   parserOptions: {
```

Comment:
Do we still need this babel parser? Can languageOptions replace this parserOptions something like 
```
languageOptions: {
      ecmaVersion: 2022,
      sourceType: 'module',
}
```